client/src/pages/login/index.tsx

```diff
@@ -52,93 +66,164 @@ function LoginPage() {
-                    value={value}
-                    onChange={onChange}
-                    data-testId="input-username"
-                  />
-                )}
-              />
-              <Controller
-                name="password"
-                control={control}
-                rules={{
-                  required: true,
-                }}
-                render={({ field: { value, onChange } }) => (
-                  <TextField
-                    id="password"
-                    label="Password"
-                    variant="outlined"
-                    type="password"
-                    value={value}
-                    onChange={onChange}
-                    data-testId="input-password"
-                  />
+    <Grid
+      container
+      style={{
+        backgroundImage: `url(${FallbackBackground})`,
+        backgroundRepeat: 'no-repeat',
+        backgroundSize: 'cover',
+        height: '100vh',
+      }}
+    >
+      <Grid item xs={0} lg={8} />
+      <Grid item xs={12} lg={4}>
+        <Box
+          sx={{
+            padding: {
+              lg: '1em',
+            },
+          }}
+          style={{ height: '100%' }}
+        >
+          <Card style={{ height: '100%' }}>
+            <CardContent
+              style={{
+                padding: '2em',
+              }}
+            >
+              <Stack spacing={2}>
+                <div>
+                  <Typography variant="h3">Sign in to Chronicle</Typography>
+                  <Typography>Enter your details below</Typography>
+                </div>
+                {error && (
+                  <>
+                    <Alert severity="error">{error}</Alert>
+                    <Spacer />
+                  </>
                 )}
-              />
-
-              <Button
-                variant="contained"
-                size="large"
-                onClick={handleSubmit(tryLogin)}
-                disabled={loading}
-                data-testId="btn-login"
-              >
-                Login
-              </Button>
-            </Stack>
-          </CardContent>
-        </Card>
-      </Box>
-    </Container>
+                <Controller
+                  name="email"
+                  control={control}
+                  rules={{
+                    required: true,
+                  }}
+                  render={({ field: { value, onChange } }) => (
+                    <TextField
+                      id="username"
+                      label="Username"
+                      variant="outlined"
+                      type="email"
+                      value={value}
+                      onChange={(e) => {
+                        handleFieldChange();
+                        onChange(e);
+                      }}
+                      onKeyDown={handleEnterKey}
+                      data-testId="input-username"
+                    />
+                  )}
+                />
```

Comment:
I would like to ask you to replace this boilerplate with @AnthonyLaiRH's `<FormInputField>` component, I believe this is a valuable improvement that is not out of scope.

Here is a usage example:

https://cs.github.com/KonomiAI/Chronicle/blob/176f38939f9c9c07395d2a41fcc4d79a97401202/client/src/pages/inventory/ProductBase.tsx?q=FormInputField#L108

smthing like:

```tsx
<FormInputField 
  name="email"
  rules={{ required: true }}
  label="Username"
  testId="input-username"
/>

Reply:
Doesn't support passwords / emails, the component is scoped to just text and numbers which makes it quite limited. Also, it doesn't allow for the existing styling to stay without updating the rest of the components which utilize `<FormInputField/>`. I think we should cut another ticket for this since it can potentially be a bigger refactor which is outside the scope of this ticket.

Reply:
Created issue here and will pick up after

https://github.com/KonomiAI/Chronicle/issues/250

Reply:
good point